en/docs/chapter_tree/binary_tree.md

```diff
@@ -221,13 +221,13 @@ The commonly used terminology of binary trees is shown in the following figure.
 
 !!! tip
 
-    Please note that we usually define "height" and "depth" as "the number of edges passed," but some problems or textbooks may define them as "the number of nodes passed." In this case, both height and depth need to be incremented by 1.
+    Please note that we typically define "height" and "depth" as "the number of edges traversed," but some problems or textbooks may define them as "the number of nodes traversed." In such cases, both height and depth need to be incremented by 1.
```

Comment:
```diff
- "the number of edges traversed,"
+ "the number of edges traversed",

- "the number of nodes traversed."
+ "the number of nodes traversed".

```

Reply:
er... my bad, I'll change it now. thanks for letting me know!